package.json

```diff
@@ -32,16 +32,17 @@
     "measure": "node ./src/utils/measure-export-time.js"
   },
   "dependencies": {
+    "@google-cloud/storage": "^6.9.2",
     "body-parser": "~1.18.3",
     "express": "~4.16.3",
     "express-request-id": "~1.4.1",
     "extract-zip": "~1.6.6",
-    "jest": "^28.1.0",
     "fs": "0.0.1-security",
     "http": "0.0.0",
     "https": "~1.0.0",
     "https-proxy-agent": "~2.2.1",
     "hummus": "~1.0.87",
+    "jest": "^28.1.0",
```

Comment:
Jest should be a dev dependency. Are there other packages that were added as regular dependencies that should not be?

Reply:
Just moved it! I didn't find any other but I'll keep it in mind.

Reply:
Done! Just moved the jest to dev dependencies now, but we'll keep it in mind.